modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -0,0 +1,105 @@
+
+image::connected-app-client-apps.png[Screenshot calling out the location of App acts on its own behalf (client credentials) and Application Creator]
+
+[calloutlist]
+. *App acts on its own behalf (client credentials)* type
+. *Add Scopes* button to add *Application Creator* scope
+
+The new Connected App can now create client applications and manage the client applications that it creates.
+
+
+== Client Application Management
+
+When a Connected App creates a client application, the owner of the Connected App is the owner of the new client application. The owner can add additional users as owners by using API Manager.   
+
+The Connected App can manage the client application it created, and can perform the following actions:
+
+* Add users as client application owners
+* Request access to an API instance
+* Request a tier change
+* Reset a client applications secret
+
+You cannot add a Connected App as an owner of a client application. Connected Apps can only manage the client applications they create. 
+
+== Creating a Client Application with the API Manager API 
+
+After creating your Connected App, you can make requests to the API Manager API to create and manage client applications. 
+
+This section only demonstrates how to create a client application. For more information about the API Manager API requests to manage client applications, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/api-manager-api/[API Manager API Documentation^].
+
+To create a client application with the API Manager API:
+
+. Collect the following information to obtain your Authorization Token:
+
+* `<connected-app-client-id>`: *Id* of your connected app
+* `<connected-app-client-secret>`: *Secret* of your connected app
+
+. Execute the following request to obtain your Authorization Token after replacing the placeholders with the following: 
++
+[source,ssh]
+----
+curl --location --request POST 'https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+ "grant_type": "client_credentials",
+ "client_id": "<connected-app-client-id>",
+ "client_secret": "<connected-app-client-secret>"
+}'
+----
+
+. Collect the following information to create your client application: 
+
+* `<your-org-id>`: *Organization ID* for the organization where you want to create the connected app
+* `<instance-id>`: API instance ID of the API instance you are creating a client application for
+* Client Application parameters:
+** `name`: Name of the application
+** `description`: Description of the application
+** `url`: URL of the application
+** `redirectURI`: An array of redirect Uris
+** `grantTypes`: An array of Grant Types, for example: `"password"`, `"implicit"`, `"client_credentials"`, `"authorization_code"`, and `"refresh_token"`
+** `apiEndpoints`: To automatically register the redirect URIs. Supported values are “true” or “false”.
```

Comment:
Not parallel with the preceding list items

Could you say "Endpoints that automatically ..."